kitsune/products/models.py

```diff
@@ -188,3 +188,71 @@ class Platform(ModelBase):
 
     def __str__(self):
         return "%s" % self.name
+
+
+class Banner(ModelBase):
+    uploaded_image = models.ImageField(
+        upload_to=settings.BANNER_IMAGE_PATH,
+        null=True,
+        blank=True,
+        max_length=settings.MAX_FILEPATH_LENGTH,
+        help_text="Must be 72x72px.",
+    )
+    static_image = models.CharField(
+        max_length=255,
+        blank=True,
+        help_text="Path to an image in staticfiles, e.g. 'protocol/img/logos/firefox/logo.svg'. "
+        "Takes precedence over uploaded_image, leave both blank to show no image.",
+    )
+    description = models.TextField()
```

Comment:
Let's make this a CharField. I highly doubt that we will need more than that for a banner. Probably we need to use a `max_length` property too to limit this in something suitable for a banner

Reply:
This is prone to be quite variable in length, given HTML is supported - I think a TextField is still more appropriate.

Reply:
Do you foresee anything that will require more than 65535 characters? I would also suggest to add a smaller max_length in the charifeild. We can start with 1024 chars or something. Charfield will help also with the validation if we ever need it